health_check/contrib/celery_beat/backends.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import (
+    ServiceReturnedUnexpectedResult, ServiceUnavailable
+)
+
+
+class CeleryBeatHealthCheck(BaseHealthCheckBackend):
+
+    def check_status(self):
+
+        try:
+            current_app = import_string('celery.current_app')
+        except ImportError:
+            self.add_error(ServiceUnavailable("No found module name 'celery.current_app'"))
+            return
+
+        try:
+            Service = import_string('celery.beat.Service')
+        except ImportError:
+            self.add_error(ServiceUnavailable("No found module name 'celery.beat.Service'"))
+            return
+
+        scheduler_module_cls = getattr(settings, 'HEALTHCHECK_CELERY_SCHEDULER', 'celery.beat.PersistentScheduler')
+        try:
+            scheduler_cls = import_string(scheduler_module_cls)
+        except ImportError:
+            self.add_error(ServiceUnavailable("No found module name '{cls}'".format(cls=scheduler_module_cls)))
+            return
```

Comment:
I don't think we need to handle those import errors. Simply let them fail. IHMO this is not an application health issue, but a runtime one and for the that an `ImportError` is perfectly suited.